cmd/stackdriver-prometheus-sidecar/main.go

```diff
@@ -104,41 +104,11 @@ func init() {
 		panic(err)
 	}
 	if err := view.Register(
-		&view.View{
-			Measure:     ocgrpc.ClientSentBytesPerRPC,
-			Name:        "grpc.io/client/sent_bytes_per_rpc",
-			Description: "Distribution of bytes sent per RPC, by method.",
-			TagKeys:     []tag.Key{ocgrpc.KeyClientMethod, ocgrpc.KeyClientStatus},
-			Aggregation: sizeDistribution,
-		},
-		&view.View{
-			Measure:     ocgrpc.ClientReceivedBytesPerRPC,
-			Name:        "grpc.io/client/received_bytes_per_rpc",
-			Description: "Distribution of bytes received per RPC, by method.",
-			TagKeys:     []tag.Key{ocgrpc.KeyClientMethod, ocgrpc.KeyClientStatus},
-			Aggregation: sizeDistribution,
-		},
-		&view.View{
-			Measure:     ocgrpc.ClientRoundtripLatency,
-			Name:        "grpc.io/client/roundtrip_latency",
-			Description: "Distribution of round-trip latency, by method.",
-			TagKeys:     []tag.Key{ocgrpc.KeyClientMethod, ocgrpc.KeyClientStatus},
-			Aggregation: latencyDistribution,
-		},
-		&view.View{
-			Measure:     ocgrpc.ClientRoundtripLatency,
-			Name:        "grpc.io/client/completed_rpcs",
-			Description: "Count of RPCs by method and status.",
-			TagKeys:     []tag.Key{ocgrpc.KeyClientMethod, ocgrpc.KeyClientStatus},
-			Aggregation: view.Count(),
-		},
-		&view.View{
-			Measure:     ocgrpc.ClientServerLatency,
-			Name:        "grpc.io/client/server_latency",
-			Description: "Distribution of server latency as viewed by client, by method.",
-			TagKeys:     []tag.Key{ocgrpc.KeyClientMethod, ocgrpc.KeyClientStatus},
-			Aggregation: latencyDistribution,
-		},
+		ocgrpc.ClientSentBytesPerRPCView,
+		ocgrpc.ClientReceivedBytesPerRPCView,
+		ocgrpc.ClientRoundtripLatencyView,
+		ocgrpc.ClientCompletedRPCsView,
+		ocgrpc.ClientServerLatencyView,
```

Comment:
I'm not sure how useful this last metric is, and it's the only one in this set that's not part of the default views that ocgrpc publishes. We could use this chance to drop it (it's easy to add it back later). What do you think?

Reply:
I don't know how useful `ocgrpc.ClientServerLatency` is either, and I don't see any reason provided in [the original commit](https://github.com/Stackdriver/stackdriver-prometheus-sidecar/commit/35ae91bb7b12cf8f3654818dca6d13def585a998) why the view was introduced, so I agree with your rationale and I am fine with dropping it.

For the reference for future readers, when we talk about default views in this thread, we are talking about [ocgrpc.DefaultClientViews](https://github.com/census-instrumentation/opencensus-go/blob/e16226179d0f7cb82bbdfed8cd824f39bb53e887/plugin/ocgrpc/client_metrics.go#L96-L102).